src/components/Header.tsx

```diff
@@ -41,13 +41,16 @@ const MenuLink = ({
   return (
     <RouteLink
       href={href}
-      color="white"
-      borderBottomWidth={isActive ? '4px' : 'none'}
-      borderColor={isActive ? 'primary.500' : 'none'}
+      color="primaryTextLightmode"
+      borderBottomWidth="4px"
+      // TODO: If link is not the active element, borderColor should equal header/drawer background
+      borderColor={isActive ? 'fuchsia.300' : 'none'}
```

Comment:
I tried your suggestion, but the non-active link elements still have a visible border line below them. Seems `none` still has some color of it own???

If the link is not active, I don't want to see any border, but it should be present (due to padding concerns).

Reply:
Ah ok, I see what you mean now. Maybe transparent might work, I need to double-check it.

Reply:
I've tried `transparent` and that does work, but now you cannot see that it's a link. 🙈 

I've decided to keep the borderColor, but make it clearer which is the active one and which the de-emphasized one.